syntax = "proto3";

package mdh.dvc.protobuf;

message StrategyConfig {
  string timestamp = 1;
  int64 id = 2;

  string value_type = 3;
  string value = 4;
  string key = 5;
  string strategy = 6;
  string parameter = 7;
  string description = 8;
  string editable = 9;
  string display = 10;
  string updated_by = 11;

  int64 updated_at = 12;
}